mobile/api_discovery.go

```diff
@@ -37,3 +38,25 @@ func DnsDiscovery(url string, nameserver string, ms int) string {
 
 	return PrepareJSONResponse(ma, nil)
 }
+
+func StartDiscoveryV5() string {
+	if wakuNode == nil {
+		return MakeJSONResponse(errWakuNodeNotReady)
+	}
+	if wakuNode.DiscV5() == nil {
+		return MakeJSONResponse(errors.New("DiscV5 is not mounted"))
+	}
+	err := wakuNode.DiscV5().Start(context.Background())
+	return MakeJSONResponse(err)
+}
+
+func StopDiscoveryV5() string {
+	if wakuNode == nil {
+		return MakeJSONResponse(errWakuNodeNotReady)
+	}
+	if wakuNode.DiscV5() == nil {
+		return MakeJSONResponse(errors.New("DiscV5 is not mounted"))
+	}
+	err := wakuNode.DiscV5().Start(context.Background())
```

Comment:
Shouldn't this be `DiscV5().Stop()`?

Reply:
:facepalm: yes

Reply:
https://github.com/waku-org/go-waku/pull/402
here's the fix